provider/vsphere/vm_template.go

```diff
@@ -138,29 +139,26 @@ func (v *vmTemplateManager) getImportedTemplate(ctx context.Context, series stri
 	}
 	logger.Tracef("Series templates: %v", seriesTemplates)
 	if len(seriesTemplates) == 0 {
-		return nil, "", errors.NotFoundf("valid template")
+		return nil, "", errors.NotFoundf("%s templates", series)
 	}
 	var vmTpl *object.VirtualMachine
 	var arch string
 	if len(arches) > 0 {
 		for _, item := range seriesTemplates {
 			arch, err = v.getVMArch(ctx, item)
 			if err != nil {
-				if errors.IsNotFound(err) {
-					continue
-				}
+				logger.Debugf("failed getVMArch for template %q: %s", item.InventoryPath, err)
```

Comment:
Do we want to ignore all errors here?
I can understand the IsNotFound but what other errors are we getting that we are essentially just turning into a Warning.

Reply:
The arch found could not be a string, or the client call to get the data failed.